### ISSUE TITLE
Target Lists - Contents too small - MSP Subscription - Trial

### ISSUE THREAD

**Issue by Braedach** [firewalla/firewalla#5699]
Target lists are too small limited at 2000 via MSP subscription.

Reference: https://help.firewalla.com/hc/en-us/articles/4409866753427#h_01FWE0P6D62DBTQCT77F89CK93

Since we are not told directly where Firewalla gets its intel from I have been making my own list from this source

https://github.com/blocklistproject/Lists#lists

I am unable to add most of these lists via Target lists as they are too large - way beyond the 2000 target size.

A number of ideas.
1.  Firewalla add these public lists via their own resources on MSP subscriptions - I am on trial.
2. Allow select lists not listed as private across the entire *.d.firewalla.org domain on a MSP subscription - we work together
3. Feedback on para 2 is already available via the android app and since deny first allow after these can be overridden by owners
4. Enable MISP integration - subject of a separate post.

The bottom line - the target lists are too small and the intel feed of Firewalla does not seem to be expanding iaw my reading on threats since well Covid (taking the easy option here).

We are all after all trying to solve the same issues and I am willing to pay for via the MSP subscription for above level protection.

I also have issues with the fact that the base Ubuntu installation is always out of date but since I only recently bridged my modem and thrown the whole lot at Firewalla am yet to see the ramifications (also a separate post).
